apps/webpack.config.js

```diff
@@ -12,6 +12,8 @@ const UnminifiedWebpackPlugin = require('unminified-webpack-plugin');
 const {WebpackManifestPlugin} = require('webpack-manifest-plugin');
 const WebpackNotifierPlugin = require('webpack-notifier');
 const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const CircularDependencyPlugin = require('circular-dependency-plugin');
+const circles = require('./circles.json');
```

Comment:
+1 for renaming the file to something like `circular_dependencies.json` to add a bit more context for the reader